community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/SchemaNumberValueConversion.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.impl.index.schema;
+
+import static org.neo4j.kernel.impl.index.schema.SchemaNumberValue.DOUBLE;
+import static org.neo4j.kernel.impl.index.schema.SchemaNumberValue.FLOAT;
+import static org.neo4j.kernel.impl.index.schema.SchemaNumberValue.LONG;
+
+/**
+ * Utilities for converting number values to and from different representations.
+ */
+class SchemaNumberValueConversion
+{
+    static void assertValidSingleNumberPropertyValue( Object[] values )
+    {
+        // TODO: support multiple values, right?
```

Comment:
Yes, that's what the line(s) below does. It's a TODO because we know that we need to support composite keys soon.